### PR TITLE
[IMP] Add filters to super_calendar

### PR DESCRIPTION
**Pull request by Ehtaga**
This PR adds features to super_calendar:
- filters (current week/month and next week/month)
- groupby (day, week, month, quarter, year)

### CODE REVIEW

**Comment by dreispt** [comment]
:+1:

**Comment by bwrsandman** [comment]
:+1:

File: super_calendar/views/super_calendar_view.xml [hunk 1]
```diff
@@ -134,6 +134,51 @@
                         <field name="date_start" />
                         <field name="duration" />
                     </group>
+
+                    <group expand="0" string="Period">
+                        <filter string="Current week"
+                                name="current_week"
+                                domain="[('date_start','&lt;=', (context_today() + relativedelta(weeks=0, weekday=6)).strftime('%%Y-%%m-%%d')),
```
Comment (lmignon) [inline super_calendar/views/super_calendar_view.xml:141]: a good candidate for https://github.com/OCA/web/blob/7.0/web_relativedelta

**Comment by lmignon** [comment]
:+1:

File: super_calendar/views/super_calendar_view.xml (hunk 1 above)
Comment (yvaucher) [inline super_calendar/views/super_calendar_view.xml:141]: This is in version 8.0 and it seems web_relativedelta was not ported

**Pull request merged**